cpp/src/groupby/sort/group_nunique.cu

```diff
@@ -87,6 +88,20 @@ struct nunique_functor {
     }
     return result;
   }
+
+  template <typename T>
+  typename std::enable_if_t<!cudf::is_equality_comparable<T, T>(), std::unique_ptr<column>>
```

Comment:
Specialization.

Reply:
Wouldn't it make more sense to use the traits to automatically handle other types as well (example, structs)?

Reply:
Agreed. I was a bit indiscriminate with my specialization comments, I didn't notice that this is using a trait instead of just specializing for `list_view`.